src/bench/bench_bitcoin.cpp

```diff
@@ -18,6 +18,8 @@
 
 static const char* DEFAULT_BENCH_FILTER = ".*";
 static constexpr int64_t DEFAULT_MIN_TIME_MS{10};
+/** Priority level default value, run "all" priority levels */
+static const std::string DEFAULT_PRIORITY_LEVEL{"all"};
```

Comment:
nit: I think `priority` better highlights the difference between the aggregated priority and the individual priority level
```suggestion
static const std::string DEFAULT_PRIORITY{"all"};
```